<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>933</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>E:/apps/Task1/icons/signal.png</normaloff>E:/apps/Task1/icons/signal.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="PlotWidget" name="graphWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>800</width>
          <height>200</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="PlotWidget" name="graphWidget2" native="true">
        <property name="minimumSize">
         <size>
          <width>800</width>
          <height>200</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string>show Spectrogram</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>200</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0" alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5" alignment="Qt::AlignHCenter">
        <widget class="QSlider" name="verticalSlider_6">
         <property name="maximum">
          <number>5</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="sliderPosition">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksAbove</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="1" alignment="Qt::AlignHCenter">
        <widget class="QSlider" name="verticalSlider_2">
         <property name="maximum">
          <number>5</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="sliderPosition">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksAbove</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="3" alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="0" column="9" alignment="Qt::AlignHCenter">
        <widget class="QSlider" name="verticalSlider_10">
         <property name="maximum">
          <number>5</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="sliderPosition">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksAbove</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="3" alignment="Qt::AlignHCenter">
        <widget class="QSlider" name="verticalSlider_4">
         <property name="maximum">
          <number>5</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="sliderPosition">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksAbove</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="6" alignment="Qt::AlignHCenter">
        <widget class="QSlider" name="verticalSlider_7">
         <property name="maximum">
          <number>5</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="sliderPosition">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksAbove</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="7" alignment="Qt::AlignHCenter">
        <widget class="QSlider" name="verticalSlider_8">
         <property name="maximum">
          <number>5</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="sliderPosition">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksAbove</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="5" alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="1" column="9" alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="0" column="8" alignment="Qt::AlignHCenter">
        <widget class="QSlider" name="verticalSlider_9">
         <property name="maximum">
          <number>5</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="sliderPosition">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksAbove</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="8" alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="1" column="7" alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4" alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2" alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4" alignment="Qt::AlignHCenter">
        <widget class="QSlider" name="verticalSlider_5">
         <property name="maximum">
          <number>5</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="sliderPosition">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksAbove</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="6" alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" alignment="Qt::AlignHCenter">
        <widget class="QSlider" name="verticalSlider">
         <property name="maximum">
          <number>5</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
         <property name="sliderPosition">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksAbove</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="2" alignment="Qt::AlignHCenter">
        <widget class="QSlider" name="verticalSlider_3">
         <property name="maximum">
          <number>5</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="sliderPosition">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksAbove</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1" rowspan="2">
     <widget class="QWidget" name="verticalWidget" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>600</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="spectroWidget2">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="spectroWidget">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="min_value_slider">
         <property name="minimum">
          <number>-200</number>
         </property>
         <property name="maximum">
          <number>0</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="min_freq">
         <property name="text">
          <string>min freq</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="max_value_slider">
         <property name="minimum">
          <number>-200</number>
         </property>
         <property name="maximum">
          <number>0</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="max_freq">
         <property name="text">
          <string>max freq</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>Palette 1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Palette 2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Palette 3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Palette 4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Palette 5</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>933</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_signal"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionSave_signal_as"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="Add_New_window"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuWindow"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="OpenSignalBtn"/>
   <addaction name="Save_signal"/>
   <addaction name="separator"/>
   <addaction name="ZoomIn"/>
   <addaction name="ZoomOut"/>
   <addaction name="separator"/>
   <addaction name="PlayBtn"/>
   <addaction name="actionSpeed_down"/>
   <addaction name="Stop"/>
   <addaction name="DrawSig"/>
   <addaction name="actionSpeed_Up"/>
   <addaction name="separator"/>
   <addaction name="AddPanel"/>
   <addaction name="PDF"/>
   <addaction name="separator"/>
   <addaction name="LeftScroll"/>
   <addaction name="RightScroll"/>
   <addaction name="separator"/>
   <addaction name="ShowFftButton"/>
  </widget>
  <action name="actionOpen_signal">
   <property name="text">
    <string>Open signal</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_signal_as">
   <property name="text">
    <string>Save signal</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Alt+F4</string>
   </property>
  </action>
  <action name="OpenSignalBtn">
   <property name="icon">
    <iconset>
     <normaloff>icons/open-folder-with-document.png</normaloff>icons/open-folder-with-document.png</iconset>
   </property>
   <property name="text">
    <string>Open Signal</string>
   </property>
   <property name="toolTip">
    <string>Open Signal</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="ZoomIn">
   <property name="icon">
    <iconset>
     <normaloff>icons/zoom-in.png</normaloff>icons/zoom-in.png</iconset>
   </property>
   <property name="text">
    <string>ZoomIn</string>
   </property>
   <property name="toolTip">
    <string>ZoomIn</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="ZoomOut">
   <property name="icon">
    <iconset>
     <normaloff>icons/zoom-out.png</normaloff>icons/zoom-out.png</iconset>
   </property>
   <property name="text">
    <string>ZoomOut</string>
   </property>
   <property name="toolTip">
    <string>ZoomOut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="PlayBtn">
   <property name="icon">
    <iconset>
     <normaloff>icons/sound.png</normaloff>icons/sound.png</iconset>
   </property>
   <property name="text">
    <string>Play signal</string>
   </property>
   <property name="toolTip">
    <string>Play signal</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="PDF">
   <property name="icon">
    <iconset>
     <normaloff>icons/pdf.png</normaloff>icons/pdf.png</iconset>
   </property>
   <property name="text">
    <string>PDF</string>
   </property>
   <property name="toolTip">
    <string>PDF</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="Spectrogram">
   <property name="text">
    <string>Spectrogram</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="Add_New_window">
   <property name="text">
    <string>Add New Window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="DeleteChannel">
   <property name="text">
    <string>Delete Channel</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="AddPanel">
   <property name="icon">
    <iconset>
     <normaloff>icons/addChannel.png</normaloff>icons/addChannel.png</iconset>
   </property>
   <property name="text">
    <string>Add panel</string>
   </property>
   <property name="toolTip">
    <string>Add Panel</string>
   </property>
  </action>
  <action name="LeftScroll">
   <property name="icon">
    <iconset>
     <normaloff>icons/left.png</normaloff>icons/left.png</iconset>
   </property>
   <property name="text">
    <string>Scroll Left</string>
   </property>
   <property name="toolTip">
    <string>Scroll Left</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Left</string>
   </property>
  </action>
  <action name="RightScroll">
   <property name="icon">
    <iconset>
     <normaloff>icons/right.png</normaloff>icons/right.png</iconset>
   </property>
   <property name="text">
    <string>Right Scroll</string>
   </property>
   <property name="toolTip">
    <string>Right Scroll</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Right</string>
   </property>
  </action>
  <action name="actionSignal2">
   <property name="text">
    <string>Signal2</string>
   </property>
  </action>
  <action name="actionSignal1_2">
   <property name="text">
    <string>Signal1</string>
   </property>
   <property name="shortcut">
    <string>1</string>
   </property>
  </action>
  <action name="actionSignal2_2">
   <property name="text">
    <string>Signal2</string>
   </property>
   <property name="shortcut">
    <string>2</string>
   </property>
  </action>
  <action name="actionSignal3">
   <property name="text">
    <string>Signal3</string>
   </property>
   <property name="shortcut">
    <string>3</string>
   </property>
  </action>
  <action name="Stop">
   <property name="icon">
    <iconset>
     <normaloff>icons/stop.png</normaloff>icons/stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>Stop</string>
   </property>
   <property name="shortcut">
    <string>F7</string>
   </property>
  </action>
  <action name="QcheckBox">
   <property name="text">
    <string>hide</string>
   </property>
   <property name="toolTip">
    <string>hide</string>
   </property>
  </action>
  <action name="DrawSig">
   <property name="icon">
    <iconset>
     <normaloff>icons/play-button.png</normaloff>icons/play-button.png</iconset>
   </property>
   <property name="text">
    <string>Draw Signal</string>
   </property>
   <property name="toolTip">
    <string>Draw Signal</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="Save_signal">
   <property name="icon">
    <iconset>
     <normaloff>icons/save.png</normaloff>icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save</string>
   </property>
  </action>
  <action name="ShowFftButton">
   <property name="icon">
    <iconset>
     <normaloff>icons/fft.png</normaloff>icons/fft.png</iconset>
   </property>
   <property name="text">
    <string>Show FFT</string>
   </property>
   <property name="toolTip">
    <string>Show FFT</string>
   </property>
  </action>
  <action name="actionSpeed_Up">
   <property name="icon">
    <iconset>
     <normaloff>icons/next.png</normaloff>icons/next.png</iconset>
   </property>
   <property name="text">
    <string>Speed Up</string>
   </property>
   <property name="toolTip">
    <string>Speed Up</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Up</string>
   </property>
  </action>
  <action name="actionSpeed_down">
   <property name="icon">
    <iconset>
     <normaloff>icons/back.png</normaloff>icons/back.png</iconset>
   </property>
   <property name="text">
    <string>Speed down</string>
   </property>
   <property name="toolTip">
    <string>Speed down</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Down</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header location="global">pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>checkBox</tabstop>
  <tabstop>verticalSlider</tabstop>
  <tabstop>verticalSlider_2</tabstop>
  <tabstop>verticalSlider_3</tabstop>
  <tabstop>verticalSlider_4</tabstop>
  <tabstop>verticalSlider_5</tabstop>
  <tabstop>verticalSlider_6</tabstop>
  <tabstop>verticalSlider_7</tabstop>
  <tabstop>verticalSlider_8</tabstop>
  <tabstop>verticalSlider_9</tabstop>
  <tabstop>verticalSlider_10</tabstop>
  <tabstop>min_value_slider</tabstop>
  <tabstop>max_value_slider</tabstop>
  <tabstop>comboBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
